python/orca/example/learn/pytorch/resnet50/inference.py

```diff
@@ -285,7 +258,6 @@ def model_creator(config):
                     model = torch.jit.trace(model, x)
                     model = torch.jit.freeze(model.eval())
                     y = model(x)
-                    y = model(x)

```

Comment:
may not remove this? Original code has this, may need to forward twice intentionally.